internal/connector/pingone/resources/sso/pingone_application.go

```diff
@@ -0,0 +1,81 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/pingidentity/pingctl/internal/connector"
+	pingoneresourcescommon "github.com/pingidentity/pingctl/internal/connector/pingone/resources/common"
```

Comment:
It doesn't look like this alias is necessary

Reply:
so, if I remove the alias, what does the call on line 35 become?

Reply:
Just "common.[etc.]", since that is the non-aliased package name

Reply:
nvm